webpack.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     linkWizard: './src/main/webapp/linkWizard/index.js',
     mainIndex: './src/main/webapp/main/main_index.js',
     search: './src/main/webapp/search/index.js',
+    project: './src/main/webapp/project/index.js'
```

Comment:
Why isn't this in feature already? It's not related to deployment ... it should already be part of the main build, but it doesn't look to be the case now. Should be a separate PR.

Reply:
You are right. I thought it was. Moving it to a separated PR.